### PR TITLE
feat: Update Inbound & Outbound API defs and typescript definitions

### PR DESCRIPTION
**Pull request by sridharvoruganti**
Closes #[2096](https://github.com/mojaloop/project/issues/2096) 

- Modify Inbound , Outbound and docker-contract API definitions accordance with new releases
- Use api-snippets typescript definitions in place of local types defined in index.d.ts (sdk-standard-components)

### CODE REVIEW

File: src/models/thirdparty.authorizations.interface.ts [hunk 1]
```diff
@@ -41,19 +44,19 @@ export enum OutboundThirdpartyAuthorizationsModelState {
...
 }
 
-export interface InboundThirdpartyAuthorizationsPutRequest {
-  challenge: string;
-  consentId: string;
-  sourceAccountId: string;
-  status: 'PENDING' | 'VERIFIED';
-  value: string;
-}
+// export interface InboundThirdpartyAuthorizationsPutRequest {
```
Comment (kleyow) [inline src/models/thirdparty.authorizations.interface.ts:47]: nit: remove?

**Review by kleyow — APPROVED** [review]
lgtm. I'd take this chance to update any snippets to v1_1 if possible, but this is a big diff so we can do that in a follow up PR if you'd like

File: src/models/thirdparty.authorizations.interface.ts (hunk 1 above)
Comment (eoln) [reply src/models/thirdparty.authorizations.interface.ts:47]: or leave doc comment

File: test/unit/handlers/inbound-api.test.ts [hunk 2]
```diff
@@ -113,17 +109,17 @@ jest.mock('~/models/inbound/authorizations.model', () => ({
...
-const putResponse: InboundAuthorizationsPutRequest = {
+const putResponse: fspiopAPI.Schemas.AuthorizationsIDPutResponse = {
   authenticationInfo: {
-    authentication: AuthenticationType.U2F,
+    authentication: 'U2F',
     authenticationValue: {
       pinValue: 'the-mocked-pin-value',
       counter: '1'
-    }
+    } as any
```
Comment (eoln) [inline test/unit/handlers/inbound-api.test.ts:118]: can't we drop `as any`  casting?

File: test/unit/models/inbound/authorizations.model.test.ts [hunk 3]
```diff
@@ -53,7 +52,7 @@ describe('InboundAuthorizationModel', () => {
       counter: '1'
```
Comment (eoln) [inline test/unit/models/inbound/authorizations.model.test.ts:52]: maybe adding the type for `authenticationValue` would prevent as from casting `as any` later?

**Review by eoln — CHANGES_REQUESTED** [review]
Great job is done @sridharvoruganti  
Few things need explanation/fixing
- why there are empty files: 
  - src/interface/inbound/api_interfaces/index.d.ts
  - src/interface/outbound/api_interfaces/index.d.ts
- please add `Sridar` to contributors :-D

File: src/models/thirdparty.authorizations.interface.ts (hunk 1 above)
Comment (sridharvoruganti) [reply src/models/thirdparty.authorizations.interface.ts]: ok

File: package.json [hunk 4]
```diff
@@ -75,7 +82,8 @@
     "@mojaloop/central-services-health": "^11.0.0",
     "@mojaloop/central-services-metrics": "^9.5.0",
     "@mojaloop/central-services-shared": "^11.5.5",
-    "@mojaloop/sdk-standard-components": "^14.1.0",
+    "@mojaloop/sdk-standard-components": "^15.0.2",
+    "@mojaloop/api-snippets": "^12.0.8",
```
[the remark this answers is not in the document]
Comment (sridharvoruganti) [reply package.json:89]: ok removed

File: package.json [hunk 5]
```diff
@@ -60,6 +67,9 @@
     "pisp"
   ],
   "author": "Paweł Marzec <pawel.marzec@modusbox.com>",
+  "contributors": [
+    "Sridhar Voruganti <sridhar.voruganti@modusbox.com>"
```
Comment (eoln) [inline package.json:71]: +1

**Review by eoln — APPROVED** [review]
lgtm

File: .circleci/config.yml [hunk 6]
```diff
@@ -260,25 +260,26 @@ jobs:
...
-          key: dependency-cache-{{ checksum "package-lock.json" }}
-      - run:
-          name: Prune non-production packages before running license-scanner
-          command: npm ci --production
-      - run:
-          name: Run the license-scanner
-          command: cd /tmp/license-scanner && pathToRepo=$CIRCLE_WORKING_DIRECTORY make run
-      - store_artifacts:
-          path: /tmp/license-scanner/results
+  # temporarily disbaled as api-snippets is not compatible with existing license policy
```
[the remark this answers is not in the document]
Comment (sridharvoruganti) [reply .circleci/config.yml:263]: ok will correct it

**Comment by sridharvoruganti** [comment]
> lgtm. I'd take this chance to update any snippets to v1_1 if possible, but this is a big diff so we can do that in a follow up PR if you'd like

it's become big PR, will do any further changes in next PR

**Pull request merged**